src/components/Header/Header.css

```diff
@@ -0,0 +1,249 @@
+a:visited {
+  color: inherit;
+}
+
+.header {
+  display: flex;
+}
+
+.header__main-top {
+  display: flex;
+  padding: 32px 0 0px;
+  border-bottom: solid;
+  border-color: rgba(207, 207, 207, 0.5);
+}
+
+.header__image-top {
+  background: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),
+    url("../../assets/ScholarSearchHubHeader.jpg");
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 100vh;
+  max-height: 576px;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  box-sizing: border-box;
+  z-index: 1;
+  color: white;
+  width: 100%;
+}
+
+.header__container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  margin: 0 40px;
+}
+
+a {
+  text-decoration: none;
+}
+
+.header__logo {
+  height: 100px;
+  width: 100px;
+}
+
+.header__title {
+  font-size: 26px;
+  font-family: "Inter Bold";
```

Comment:
It's important to always provide **fallback** and **generic** font name for `font-family`, like that: 

```css

font-family: 'Inter', Arial, sans-serif; 


```

https://developer.mozilla.org/en/docs/Web/CSS/font-family

Font names are just for an example, you have your own font names and need to set fallback fonts based on it. For example, `Noto Serif` and `Inter` belong to different types, and they need different fallback fonts.
https://www.w3schools.com/css/css_font.asp 

Please, make this correction for **all the places** where you use `font-family`